app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -136,6 +136,7 @@ public void onResponse(Call<ForgotPasswordResponse> call, Response<ForgotPasswor
                                 public void onClick(DialogInterface dialog, int id) {
                                     Intent intent = new Intent(ForgotPasswordActivity.this, LoginActivity.class);
                                     startActivity(intent);
+                                    finish();
```

Comment:
@amitiwary999 When login activity is started from forgot password activity, we have to press back twice to exit app. Pressing back once opens previous instance of login activity. Either use intent flags `intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);` or override onBackPressed method in login activity and call finishAffinity().